ext/posix/posix.c

```diff
@@ -123,6 +125,7 @@ ZEND_GET_MODULE(posix)
 
 PHP_FUNCTION(posix_kill)
 {
+#ifndef __wasi__
```

Comment:
It is more common to make these functions not available is they don't work, instead of them just returning a single value. This way, user code can check whether they can use the function with `function_exists`, which they can't do with your change here.

Reply:
Updated to reflect feedback; thanks!

Reply:
Hi @ereslibre I don't think you have. For example at https://github.com/php/php-src/pull/10457/files#diff-28ed31fa6b0d63b5c77f4c164e93fc6b0057d286d607c0d8d73897f5bd66bb6cR798, the function is still available, but always returns the same value. These functions should not be available at all if they don't do anything.

Reply:
Sorry, you are right. I am updating the PR.